DEVELOPERS.md

```diff
@@ -166,4 +165,18 @@ A detailed explanation can be found in this [document][commit-message-format].
 [yarn-install]: https://yarnpkg.com/en/docs/install
 
 
+## <a name="troubleshooting"> Troubleshooting
+
+#### "Too Many Open Files" on OS X
+
+If while running the test suite you get failing tests with "too many open files", you may need to adjust the number of files your shell process is willing to open. This is particularly common with the `p2p` and `cmd` pacakges. Both rely heavily on interacting temporary directories on the file system to run tests.
+
+Often this is caused by having too low a `ulimit` value for the shell. Try the following command:
+
+```
+ulimit -S -n 1000
+```
+
+We recommend a ulimit value of at least 1000, feel free to go higher if you need. Some other software require values as high as 10000 to avoid max open file issue.
```

Comment:
```suggestion
We recommend a ulimit value of at least 1000; feel free to go higher if you need. Some other software require values as high as 10000 to avoid the max open file issue.
```